corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -119,7 +120,11 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
             var id = model.get('id'),
                 inputId = id + "_mapbox",
                 $field = $("#" + inputId);
-
+            $(function () {
+                $field.on("change", function() {
+                    kissmetrics.track.event("Accessibility Tracking - Geocoder Interaction in Case Seach");
```

Comment:
`initMapboxWidget` gets called within `initGeocoders`, which makes me think this event will always fire when there's a geocoder on the page, not just when there's interaction. `geocoderItemCallback` is probably the function to hook into, since that's the one that broadcasts when the geocoder gets a new value (and maybe also `geocoderOnClearCallback`).